core/src/test/java/cz/xtf/core/openshift/mocked/smoke/BasicOpenShiftTest.java

```diff
@@ -0,0 +1,102 @@
+package cz.xtf.core.openshift.mocked.smoke;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import cz.xtf.core.openshift.OpenShift;
+import io.fabric8.kubernetes.api.model.ConfigMap;
+import io.fabric8.kubernetes.api.model.ConfigMapBuilder;
```

Comment:
shouldn't it rather test API from XTF? cz.xtf.builder.builders.ConfigMapBuilder? DeploymentConfigBuilder? ...?

Reply:
+1 @simkam - here I just sketched the basic idea, I'll take your advice into account for the changes I'll be doing next.

Reply:
Hi @simkam - I pushed changes following your suggestion. Let me know whether this could be eough for a baseline. In case you'd want more tests (i.e. more builders) then I'll keep it as a draft and add them in the next days.

Reply:
looks good to me